hadoop.yaml

```diff
@@ -0,0 +1,86 @@
+package:
+  name: hadoop
+  version: 3.4.0
+  epoch: 0
+  description: Open Source Identity and Access Management For Modern Applications and Services
+  copyright:
+    - license: Apache-2.0
+  target-architecture:
+    # note: skipping aarch64 because frontend-maven-plugin doesn't work on aarach arch.
+    - x86_64
+  dependencies:
+    runtime:
+      - openjdk-8
+
+environment:
+  contents:
+    packages:
+      - bash
+      - busybox
+      - ca-certificates-bundle
+      - maven
+      - nodejs
+      - openjdk-8
+      - wolfi-base
+      - wolfi-baselayout
+      - yarn
+  environment:
+    LANG: en_US.UTF-8
+    JAVA_HOME: /usr/lib/jvm/java-1.8-openjdk
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/apache/hadoop
+      tag: rel/release-${{package.version}}
+      expected-commit: bd8b77f398f626bb7791783192ee7a5dfaeec760
+
+  - uses: maven/pombump
+    with:
+      patch-file: pombump-deps.yaml
+      properties-file: pombump-properties.yaml
+      pom: hadoop-project/pom.xml
+
+  - runs: |
+      tar --version
+      mkdir -p ${{targets.contextdir}}/
```

Comment:
The destination directory (destdir) already exists. We only need to do this for subpackages. Not to mention, this is handled by the command below it